### PR TITLE
Fix report image issues [SCI-3772]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-3772](https://biosistemika.atlassian.net/browse/SCI-3772)

### What was done
Fix report image issues

### CODE REVIEW

File: app/models/asset.rb [hunk 1]
```diff
@@ -448,6 +449,10 @@ def editable_image?
     !locked? && %r{^image/#{Regexp.union(Constants::WHITELISTED_IMAGE_TYPES_EDITABLE)}} =~ file.content_type
   end
 
+  def generate_base64(style)
```
Comment (radiokills) [inline app/models/asset.rb]: Why are we even passing style here?

File: app/services/reports/docx/draw_step_asset.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ def draw_step_asset(subject)
     asset = Asset.find_by_id(subject['id']['asset_id'])
     return unless asset
 
-    is_image = asset.is_image?
+    is_image = asset.image?
```
Comment (radiokills) [inline app/services/reports/docx/draw_step_asset.rb]: Do you really need new variable here?

File: app/models/user.rb [hunk 3]
```diff
@@ -573,19 +574,12 @@ def file_name
...
 
-    avatar_uri = if avatar.options[:storage].to_sym == :s3
-                   URI.parse(avatar.url(style)).open.to_a.join
-                 else
-                   File.open(avatar.path(style)).to_a.join
-                 end
-
-    encoded_data = Base64.strict_encode64(avatar_uri)
-    "data:#{avatar_content_type};base64,#{encoded_data}"
+    convert_variant_to_base64(avatar_variant(style))
```
Comment (radiokills) [inline app/models/user.rb]: Nice! :+1:

File: app/models/asset.rb (hunk 1 above)
Comment (aignatov-bio) [reply app/models/asset.rb]: Currently we use it only in one place, but in future we can use it same as avatars, with multiple styles.

File: app/services/reports/docx/draw_step_asset.rb (hunk 2 above)
Comment (aignatov-bio) [reply app/services/reports/docx/draw_step_asset.rb]: Fixed

**Pull request merged**